src/hapi/pipelines/configs/food_security.yaml

```diff
@@ -1,75 +1,36 @@
-# IPC name and description taken from page 53 of
-# https://www.ipcinfo.org/fileadmin/user_upload/ipcinfo/manual/IPC_Technical_Manual_3_Final.pdf
+food_security:
+  adm_ignore_patterns:
+    - " urban"
+    - " rural"
+    - "rural "
+    - "(1)"
+    - "(2)"
+    - "(3)"
+    - "(4)"
+    - "(5)"
+    - "_1"
+    - "_2"
+    - "_3"
+    - "_4"
+    - "idp"
+    - "refugee"
+    - "returnee"
+    - " cluster "
+    - " commune "
 
-food_security_default:
-  scrapers_with_defaults:
-    - "food_security_ch"
-  input:
-    - "chtype"
-    - "reference_label"
-    - "reference_year"
-    - "population"
-    - "phase1"
-    - "phase2"
-    - "phase3"
-    - "phase4"
-    - "phase5"
-    - "phase35"
-  list:
-    - "chtype"
-    - "reference_label"
-    - "reference_year"
-    - "population"
-    - "phase1"
-    - "phase2"
-    - "phase3"
-    - "phase4"
-    - "phase5"
-    - "phase35"
-  output:
-    - "ipc_type"
-    - "reference_period_months"
-    - "reference_period_year"
-    - "population_total"
-    - "population_phase1"
-    - "population_phase2"
-    - "population_phase3"
-    - "population_phase4"
-    - "population_phase5"
-    - "population_phase3+"
-  output_hxl:
-    - "#ipc+type"
-    - "#date+months"
-    - "#date+years"
-    - "#affected+ch+food+analysed+num"
-    - "#affected+ch+food+p1+num"
-    - "#affected+ch+food+p2+num"
-    - "#affected+ch+food+p3+num"
-    - "#affected+ch+food+p4+num"
-    - "#affected+ch+food+p5+num"
-    - "#affected+ch+food+p3plus+num"
+  adm1_only:
+    - "HTI"
+    - "MMR"
 
-food_security_adminone:
-  food_security_ch:
-    dataset: "cadre-harmonise"
-    format: "xlsx"
-    admin:
-      - "adm0_pcod3"
-      - "adm1_pcod2"
-    admin_exact: False
-    prefilter: "adm0_pcod3 in ['AFG', 'BFA', 'CAF', 'CMR', 'COD', 'COL', 'ETH', 'GTM', 'HND', 'HTI', 'MLI', 'MMR', 'MOZ', 'NER', 'NGA', 'PSE', 'SDN', 'SLV', 'SOM', 'SSD', 'SYR', 'TCD', 'UKR', 'VEN', 'YEM'] and adm2_pcod2 is None"
-    filter_cols:
-      - "adm0_pcod3"
-      - "adm2_pcod2"
+  adm2_only:
+    - "PSE"
 
-food_security_admintwo:
-  food_security_ch:
-    dataset: "cadre-harmonise"
-    format: "xlsx"
-    admin:
-      - "adm0_pcod3"
-      - "adm2_pcod2"
-    admin_exact: False
-    prefilter: "adm0_pcod3 in ['AFG', 'BFA', 'CAF', 'CMR', 'COD', 'COL', 'ETH', 'GTM', 'HND', 'HTI', 'MLI', 'MMR', 'MOZ', 'NER', 'NGA', 'PSE', 'SDN', 'SLV', 'SOM', 'SSD', 'SYR', 'TCD', 'UKR', 'VEN', 'YEM']"
-    filter_cols:
-      - "adm0_pcod3"
+  adm1_errors:
+    - "Distrito Central"  # adm2 not adm1
+
+  adm2_errors:
```

Comment:
How did you come up with this list, is it possible that there are more? Is it something we have to be careful with when the IPC data is updated?

Reply:
I think I'm seeing some similar issues in the WFP data, where 1) our admin boundaries are different versions or 2) our admin level definitions are different, e.g. WFP has region for adm1 and we use district.  We'll have to be careful with that data as well, I was thinking of making the fuzzy matching more strict.

Reply:
Yes we will need to be careful. They use "Level 1" and "Area" loosely